main/src/library/Nel.flix

```diff
@@ -79,10 +79,24 @@ instance Iterable[Nel] {
     pub def iterator(t: Nel[a]): Iterator[a] & Impure =
         Nel.toIterator(t)
 }
-
 instance Reducible[Nel] {
     pub def reduceLeftTo(f: (b, a) -> b & ef1, g: a -> b & ef2, l: Nel[a]): b & (ef1 and ef2) = Nel.reduceLeftTo(f, g, l)
     pub def reduceRightTo(f: (a, b) -> b & ef1, g: a -> b & ef2, l: Nel[a]): b & (ef1 and ef2) = Nel.reduceRightTo(f, g, l)
+    override pub def head(l: Nel[a]): a = Nel.head(l)
+    override pub def last(l: Nel[a]): a = Nel.last(l)
+    override pub def init(l: Nel[a]): List[a] = Nel.init(l)
+    override pub def tail(l: Nel[a]): List[a] = Nel.tail(l)
+    override pub def exists(f: a -> Bool & ef, l: Nel[a]): Bool & ef = Nel.exists(f, l)
```

Comment:
I understand overriding the above signatures, but why override e.g. memberOf?

Reply:
Same as `exists` and `forall`; to allow early termination